.github/workflows/haskell.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   build:
-    name: GHC ${{ matrix.name }}
+    name: GHC ${{matrix.variant}}
```

Comment:
Why this change? Why not just use `name`?

Reply:
I did more changes earlier where name was derived from variant, ghc version and other things. Those changes were reverted but I kept the variant.